apps/clamav.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
+
+# T&M Hansson IT AB © - 2020, https://www.hanssonit.se/
+
+# shellcheck disable=2034,2059
+true
+SCRIPT_NAME="ClamAV"
+SCRIPT_EXPLAINER="This script installs the open-source Antivirus-software ClamAV on your server \
+and configures Nextcloud to detect infected files already during the upload.
+At the end of the script, you will have the chance to setup a weekly full-scan of all files."
+# shellcheck source=lib.sh
+. <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh)
+
+# Check for errors + debug code and abort if something isn't right
+# 1 = ON
+# 0 = OFF
+DEBUG=0
+debug_mode
+
+# Check if root
+root_check
+
+# Show explainer
+explainer_popup
+
+# Needs 1 GB alone
+ram_check 3 "ClamAV"
```

Comment:
Why do we need 1 extra GB here?

Reply:
Because the ClamAV-deamon alone needs 1GB and since php-fpm is recommended to run with at least 2GB of available RAM, I though this would make sense.

Reply:
OK :+1: